src/core/xconnection.rs

```diff
@@ -474,13 +523,13 @@ impl XcbConnection {
     fn window_has_type_in(&self, id: WinId, win_types: &Vec<u32>) -> bool {
         // xcb docs: https://www.mankier.com/3/xcb_get_property
         let cookie = xcb::get_property(
-            &self.conn,                             // xcb connection to X11
-            false,                                  // should the property be deleted
-            id,                                     // target window to query
-            self.known_atom("_NET_WM_WINDOW_TYPE"), // the property we want
-            xcb::ATOM_ANY,                          // the type of the property
-            0,                                      // offset in the property to retrieve data from
-            2048,                                   // how many 32bit multiples of data to retrieve
+            &self.conn,                              // xcb connection to X11
+            false,                                   // should the property be deleted
+            id,                                      // target window to query
+            self.known_atom(&Atom::NetWmWindowType), // the property we want
```

Comment:
this should just be a `get` from the hashmap

Reply:
Does this mean that `atom` and `known_atom` are no longer needed?